test/test_suit/test_schema_mapping.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, driver, nameSalt):
         self.gold = {
             'PERFORMANCE_STRING': 'Excellent',
             'case_sensitive_PERFORMANCE_CHAR': 'A',
-            'PERFORMANCE_HEX': b'\xef\xbf\xbd\xef\xbf\xbd\xef\xbf\xbd\xef\xbf\xbd',
+            'PERFORMANCE_HEX': b'\xff\xff\xff\xff',
```

Comment:
@sfc-gh-lsembera any idea which change from 1.0.3-beta to 1.1.0 cause this behavior change? The new value looks right to me though.....